### PR TITLE
Php form

### PR DESCRIPTION
**Pull request by nburd90**
Added php to echo input.

### CODE REVIEW

**Comment by billythakidd04** [comment]
before i get too deep into this there is no description and none of your team are added as reviewers

**Review by tfrazzini13 — APPROVED** [review]

File: contact.php [hunk 1]
```diff
@@ -4,20 +4,30 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"/>
```
Comment (billythakidd04) [inline contact.php:8]: strange choice for bootstrap, why this one?

**Review by billythakidd04 — COMMENTED** [review]
Why did we use that version of bootstrap? isnt everything else on 5?

**Comment by nburd90** [comment]
Must have copied the wrong one. Updating now.

**Pull request merged**